### PR TITLE
Alexaustin/h4 ish 4 food pantry component

### PR DESCRIPTION
**Pull request by raustin9**
Created the card component that has the information of each food pantry that will be displayed on the site. It has the phone number, address, name, and hours. I also added 'react/prop-types': 'off' to line 39 of the .eslintrc.js file so that we do not run into any errors with props-type because typescript handles it already.

### CODE REVIEW

File: src/components/FoodPantryCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent } from 'react';
+import { FoodPantry } from 'utils/types';
+
+interface Props {
+  pantry: FoodPantry;
+}
+
+const FoodPantryCard: FunctionComponent<Props> = ({ pantry }) => {
```
Comment (VishalAiely) [inline src/components/FoodPantryCard/index.tsx:8]: I think leaving them as h1 tags for now just to mark all the info in the pattern we need is good. We can come back to styling once we make some progress on design

File: src/components/FoodPantryCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <div>
+      <h1 id="pantry_name">Name: {pantry.name}</h1>
+      <h1 id="pantry_address">
+        Address: {pantry.address.streetNumber} {pantry.address.streetName}, {pantry.address.zipCode}
+      </h1>
+      <h1 id="hours">
+        Hours: Monday - {pantry.hours.monday} Tuesday - {pantry.hours.tuesday} Wednesday - {pantry.hours.wednesday}{' '}
+        Thusday - {pantry.hours.thursday} Friday - {pantry.hours.friday} Saturday - {pantry.hours.saturday} Sunday -{' '}
+        {pantry.hours.sunday}
```
Comment (VishalAiely) [inline src/components/FoodPantryCard/index.tsx:18]: A small recommendation would be to separate each of these days on a separate line to make it a bit more readable, but don't worry about it for this PR

**Review by VishalAiely — APPROVED** [review]
Overall the code is concise, readable, and contains all the info for the pantry card. Good work 👍

File: src/components/FoodPantryCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import { FoodPantry } from 'utils/types';
+
+interface Props {
+  pantry: FoodPantry;
+}
+
+const FoodPantryCard: FunctionComponent<Props> = ({ pantry }) => {
+  return (
+    <div>
+      <h1 id="pantry_name">Name: {pantry.name}</h1>
```
Comment (braxtonhaynie) [inline src/components/FoodPantryCard/index.tsx:11]: I noticed that there is an option for a website in types.ts. Could look at possibly adding that to the card as well.

**Review by braxtonhaynie — APPROVED** [review]
I think it all looks good overall. I agree with the comments vishal made, but they aren't big issues and can be revisited later.

**Pull request merged**